src/map/index.js

```diff
@@ -10,22 +10,35 @@ const map = new mapboxgl.Map({
   style: 'mapbox://styles/mapbox/light-v10',
   center: [115.817465, -31.976629],
   zoom: 17,
-  pitchWithRotate: false,
-  dragRotate: false
+  pitchWithRotate: true,
+  dragRotate: true
 })
 
+var popupOffsets = {
+  top: [100, 0],
+  'top-left': [100, 0],
+  'top-right': [100, 0]
+}
+
 const currentPos = document.createElement('div')
 currentPos.id = 'current-marker'
-const currentMarker = new mapboxgl.Marker(currentPos)
+const currentMarker = new mapboxgl.Marker(currentPos, {
+  offset: [15, -30]
+})
   .setLngLat([locations[0].positions.lon, locations[0].positions.lat])
   .addTo(map)
 
+var nav = new mapboxgl.NavigationControl()
```

Comment:
```suggestion
const nav = new mapboxgl.NavigationControl()
```
also consider using a more explicit name like navControl or mapControl